src/views/CreateTransformer.vue

```diff
@@ -0,0 +1,177 @@
+<template>
+  <main v-if="formState === 0" @keydown.enter="nextStep">
+    <ViewHeader title="How should the new transformer be called?" :backLink="{ link: { name: 'transformers' } }"/>
+    <Input :value.sync="newTransformer.name" name="Name" placeholder="Filter for today" :autofocus="true"/>
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 1">
+    <ViewHeader title="Which resource type should be transformed?" :backLink="{ onClick: previousStep }" />
+    <ListSection :list="resourceTypesList"/>
+  </main>
+
+  <main v-else-if="formState === 2">
+    <ViewHeader title="Please choose the type of the transformer." :backLink="{ onClick: previousStep }" />
+    <Select
+          :value.sync="newTransformer.transformerType"
+          name="Type of the transformer"
+          :required="true"
+          :options="transformerTypeOptions"
+        />
+    <Button text="Continue" :onClick="nextStep"/>
+  </main>
+
+  <main v-else-if="formState === 3">
+    <ViewHeader title="Please insert the functionality of the transformer" :backLink="{ onClick: previousStep }" />
+    <ListSection
+      :title="`Attributes of ${newTransformer.resourceType.name}`"
+      :list="resourceTypeAttributeList"
+    />
+    <br>
+    <h1>below you can enter the code of your transformer</h1>
+    <br>
+    <TextArea
+      :label="`instancesOf${this.nameWithoutWhitespace}.${newTransformer.transformerType}( (instance) => {`"
+      footerLabel='});'
+      :value.sync="newTransformer.body"
+      placeholder="var date = new Date();
+  var currentYear = date.getFullYear();
```

Comment:
Is this indentation needed for layout? Or can the text be indented?
Maybe its better to extract this long string to a constant variable?